client/packages/common/src/intl/locales/ru/common.json

```diff
@@ -947,8 +947,8 @@
   "label.pack-variants": "Варианты Упаковок",
   "label.packs": "Упаковки",
   "label.packs-available": "Доступные упаковки",
-  "label.packs-of-1": "{{count}} {{unitWord}} в упаковках по 1 штуке",
-  "label.packs-of-size": "{{packs}} упаковок по {{size}} {{unitWord}} ({{count}} единицы)",
+  "label.packs-of-1": "{{numUnits}} {{unitType}} в упаковках по 1 штуке",
+  "label.packs-of-size": "{{numPacks}} {{packWord}} {{packSize}} {{unitType}} ({{numUnits}} {{unitWord}})",
```

Comment:
I would be a little careful updating the non-english translations without knowing the language. We can't be sure that the structure of this sentence is in the correct order in Russian. Normally it's best to just update the English file and let the Weblate automation take care of the rest (it'll mark them as needing updating by translators).

In this case, I'd leave it for now, but might pay to mention it to the Russian, French & Spanish speakers in the office and get them to double-check this in their language when testing.